docs/tutorials-and-quickstart-guides/openms-user-tutorial.md

```diff
@@ -0,0 +1,3023 @@
+
+Dependent on your data MS/MS spectra can be visualized as well (see Fig.5) . You can
+do so, by double-click on the MS/MS spectrum shown in scan view
+
+|![ms2 spectrum](../images/openms-user-tutorial/introduction/ms2_introduction.png)|
+|:--:|
+|Figure 5: MS/MS spectrum|
+
+### Introduction to KNIME/OpenMS
+
+Using OpenMS in combination with KNIME, you can create, edit, open, save, and run workflows that combine TOPP tools with
+the powerful data analysis capabilities of KNIME. Workflows can be created conveniently in a graphical user interface.
+The parameters of all involved tools can be edited within the application and are also saved as part of the workflow.
+Furthermore, KNIME interactively performs validity checks during the workflow editing process, to make it more
+difficult to create an invalid workflow. Throughout most parts of this tutorial, you will use KNIME to create and execute
+workflows. The first step is to become familiar with KNIME. Additional information on the basic usage of KNIME can be
+found on the KNIME [Getting Started page](https://www.knime.com/getting-started-guide). However, the most important
+concepts will also be reviewed in this tutorial.
+
+#### Plugin and dependency installation
+
+Before we can start with the tutorial, we need to install all the required extensions for KNIME. Since KNIME 3.2.1, the
+program automatically detects missing plugins when you open a workflow but to make sure that the right source for the
+OpenMS plugin is chosen, please follow the instructions here. First, we install some additional extensions that are
+required by our OpenMS nodes or used in the Tutorials e.g. for visualization and file handling.
+
+1. Click on **Help** > **Install New Software**.
+2. From the **Work with**: drop-down list select http://update.knime.com/analytics- platform/4.4.
+3. Now select the following plugins from the KNIME & Extensions category
+   - KNIME Base Chemistry Types & Nodes
+   - KNIME Chemistry Add-Ons
+   - KNIME File Handling Nodes (required for OpenMS nodes in general)
+   - KNIME Interactive R Statistics Integration
+   - KNIME Report Designer
+   - KNIME SVG Support
+4. Click on **Next** and follow the instructions (you may but don’t need to restart KNIME now).
+5. Click again on **Help** > **Install New Software**
+6. From the **Work with**: drop-down list select http://update.knime.com/community -contributions/trusted/4.4
+7. Now select the following plugin from the ”KNIME Community Contributions - Cheminformatics” category
+   - RDKit KNIME integration
+8. Click on **Next**  and follow the instructions and after a restart of KNIME the dependencies will be installed.
+
+In addition, we need to install R for the statistical downstream analysis. Choose the directory that matches your
+operating system, double-click the R installer and follow the instructions. We recommend to use the default settings
+whenever possible. On macOS you also need to install XQuartz from the same directory.
```

Comment:
This assumes there is an USB stick with those installers, right? Which versions of R are supported, where do you download them? A description of what to do without the USB stick would be nice.
R 3.6.X is mentioned, is that the version range our workflows support?

Reply:
I remember I had some trouble a year ago with installing the necessary packages further below with R 4, and R 3.6.3 is the highest version before that.
This is probably the highest version supported by the R scripts in our KNIME workflows: https://cran.r-project.org/bin/windows/base/old/3.6.3/

Can anyone else confirm or correct that?

Reply:
Would you be able to provide an alternative?